AuditEventProcessor.cpp

```diff
@@ -47,6 +47,8 @@ extern "C" {
 
 #define PROCESS_CREATE_RECORD_TYPE 14688
 #define PROCESS_CREATE_RECORD_NAME "AUOMS_EXECVE"
+#define FRAGMENT_RECORD_TYPE 11309
```

Comment:
What is the rationale behind these random record type numbers???

Reply:
So, the rationale is that an EXECVE record type is 1309, so I went for 11309 for the fragment records, as these are essentially parsed EXECVEs. The 14688 is a nod to windows 4688 events (process create) but I'm happy to pick different numbers if these seem inappropriate?

Reply:
Please add a comment for each explaining the rational for the numbers. Also, indicate that they are in the 10000 range so that we void possible collision with official audit record type codes.